
#include "game_client_player_handler.h"

///<<< BEGIN WRITING YOUR CODE
///<<< END WRITING YOUR CODE



void GameClientPlayerCommonServiceHandler::SendTipToClient(entt::entity player,const ::TipInfoMessage* request,
	::Empty* response)
{
///<<< BEGIN WRITING YOUR CODE
///<<< END WRITING YOUR CODE


}



void GameClientPlayerCommonServiceHandler::KickPlayer(entt::entity player,const ::GameKickPlayerRequest* request,
	::Empty* response)
{
///<<< BEGIN WRITING YOUR CODE
///<<< END WRITING YOUR CODE


}


